src/display/editor/tools.js

```diff
@@ -919,6 +919,14 @@ class AnnotationEditorUIManager {
    */
   paste(event) {
     event.preventDefault();
+    const { clipboardData } = event;
+    for (const item of clipboardData.items) {
+      for (const editorType of this.#editorTypes) {
+        if (editorType.isHandlingMimeForPasting(item.type)) {
+          editorType.paste(item, this.currentLayer);
+        }
+      }
+    }
 
     let data = event.clipboardData.getData("application/pdfjs");
```

Comment:
Nit: Given the code above, this can be now slightly shorted.
```suggestion
    let data = clipboardData.getData("application/pdfjs");
```